Main/DateColumn.h

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2017. All rights reserved.
+ * Distributed under the terms of the MIT license.
+ *
+ * Authors:
+ *		Janus
+ */
+#ifndef DATE_COLUMN_H
+#define DATE_COLUMN_H
+
+#include <ColumnTypes.h>
+
+class DateColumn : public BDateColumn
+{
+	public:
+					DateColumn	(const char* title,
+								 float width,
+								 float minWidth,
+								 float maxWidth,
+								 alignment align = B_ALIGN_LEFT);
+		  void		DrawField	(BField* field,
+								 BRect rect,
+								 BView* parent);
+};
```

Comment:
The indentation in the rest or PecoRename's .h are different. It's probably not "real" Haiku code style, but for consistency's sake...